multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangareader/WPMangaReaderGenerator.kt

```diff
@@ -23,7 +23,7 @@ class WPMangaReaderGenerator : ThemeSourceGenerator {
         SingleLang("Gabut Scans", "https://gabutscans.com", "id"),
         SingleLang("Martial Manga", "https://martialmanga.com/", "es"),
         SingleLang("Ngomik", "https://ngomik.net", "id", overrideVersionCode = 1),
-        SingleLang("Sekaikomik", "https://www.sekaikomik.in", "id", isNsfw = true, overrideVersionCode = 7),
+        SingleLang("Sekaikomik", "https://www.sekaikomik.site", "id", isNsfw = true, overrideVersionCode = 7),
```

Comment:
```suggestion
        SingleLang("Sekaikomik", "https://www.sekaikomik.site", "id", isNsfw = true, overrideVersionCode = 8),
```

Need to bump the `overrideVersionCode`.